web/src/app/modules/shared/components/presentation/select-file/select-file.component.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2021 the Octant contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  isDevMode,
+  OnInit,
+  Output,
+  ViewChild,
+} from '@angular/core';
+import { AbstractViewComponent } from '../../abstract-view/abstract-view.component';
+import { SelectFileView } from '../../../models/content';
+import { WebsocketService } from '../../../../../data/services/websocket/websocket.service';
+import { ActionService } from '../../../services/action/action.service';
+
+@Component({
+  selector: 'app-view-select-file',
+  templateUrl: './select-file.component.html',
```

Comment:
I wonder if it should switch the change detection to "ChangeDetectionStrategy.OnPush"

Reply:
I was considering it, but since it’s a simple component, I decided to keep it simple for now.